persistence/test/setup_test.go

```diff
@@ -204,10 +206,11 @@ func fuzzSingleProtocolActor(
 			require.NoError(t, err)
 
 			require.ElementsMatch(t, newActor.Chains, newChains, "staked chains not updated")
+			require.NotContains(t, newActor.StakedTokens, "invalid")
 			// TODO(andrew): Use `require.Contains` instead
-			if strings.Contains(newActor.StakedTokens, "invalid") {
-				fmt.Println("")
-			}
+			// if strings.Contains(newActor.StakedTokens, "invalid") {
```

Comment:
Why is this commented out?

Also, if you're going to touch this code, it seems like you could easily satisfy that TODO there.

Perhaps it's out of scope?